pkg/dao/repositories.go

```diff
@@ -66,11 +71,17 @@ func (p repositoryDaoImpl) FetchForUrl(url string) (Repository, error) {
 	return internalRepo, nil
 }
 
-func (p repositoryDaoImpl) List() ([]Repository, error) {
+func (p repositoryDaoImpl) List(ignoreFailed bool) ([]Repository, error) {
 	var dbRepos []models.Repository
 	var repos []Repository
 	var repo Repository
-	result := p.db.Find(&dbRepos)
+	var result *gorm.DB
+
+	if ignoreFailed {
+		result = p.db.Where("failed_introspections_count < ?", config.FailedIntrospectionsLimit+1).Find(&dbRepos)
```

Comment:
I made this  `config.FailedIntrospectionsLimit+1` so that it errors the first time the limit is hit, but after that it skips.